### PR TITLE
[website] update downlink used for quickstart

### LINKED ISSUE #2153: Update pulsar functions SDK documentation
#2137 update some python function examples. we need to make sure documentation page are updated.

### PR DESCRIPTION
**Pull request by sijie**
### Changes

- Use archieve links to ensure all command line works with old releases
- Fix formatting
- Remove redundant pip dependencies

Also Fixes #2153

### CODE REVIEW

File: site2/website/scripts/replace.js [hunk 1]
```diff
@@ -22,11 +22,11 @@ function downloadPageUrl() {
 }
 
 function binaryReleaseUrl(version) {
-  return `http://www.apache.org/dyn/closer.cgi/incubator/pulsar/pulsar-${version}/apache-pulsar-${version}-bin.tar.gz`
```
Comment (ivankelly) [inline site2/website/scripts/replace.js]: why are you making this change? I suspect the asf ops folks would frown on this.

Comment (sijie) [reply site2/website/scripts/replace.js]: archive is okay, it just can't be dist directly. all the links in our download pages for old releases are from archive.

closer.cgi is a redirection to a page showing what is the closer mirror. but it doesn't download directly. so when it is used in command line,  it doesn't really work. I noticed this when I am trying out all the quickstart in pulsar documentation. so this leads me to change it to archive. do you know any better links here?

Comment (ivankelly) [reply site2/website/scripts/replace.js]: ah, that does suck.

**Comment by sijie** [comment]
run java8 tests

**Comment by sijie** [comment]
run cpp tests

**Comment by srkukarni** [comment]
run java8 tests

**Comment by sijie** [comment]
run java8 tests

**Pull request merged**